src/components/modals/AddQuestion/AddQuestion.jsx

```diff
@@ -6,17 +6,18 @@ import * as Yup from 'yup';
 import Dropzone from 'react-dropzone';
 import { FaFileUpload, FaSpinner } from 'react-icons/fa';
 import prettyBytes from 'pretty-bytes';
-
+import { useState } from 'react';
 import Modal from '../Modal/Modal';
 import FormField from '../../FormField/FormField';
 
 const SignInSchema = Yup.object().shape({
   title: Yup.string().min(3).max(24).required(),
   question: Yup.string().email().required(),
-  attachment: Yup.string().min(8).max(32).required(),
 });
 
 function AddQuestion({ open, setOpen }) {
+  // eslint-disable-next-line no-unused-vars
+  const [newPic, setNewPic] = useState('');
```

Comment:
What is the unused variable?

Reply:
This is what we gonna use to store the pic locally before dispatching them with the form before dispatching the question. Let me know if you want me to remove them for now.